app/controllers/api/v1/positions_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Api::V1
+  # Controller for Positions
+  class PositionsController < ApplicationController
+    before_action :set_position, only: %i[show update destroy]
+
+    # GET /positions
+    def index
+      @positions = Position.order(:id)
+
+      render json: @positions
+    end
+
+    # GET /positions/1
+    def show
+      render json: @position
+    end
+
+    # POST /positions
+    def create
+      @position = Position.new(position_params)
+
+      if @position.save
+        render json: @position, status: :created
+      else
+        render json: @position.errors, status: :unprocessable_entity
+      end
+    end
+
+    # PATCH/PUT /positions/1
+    def update
+      if @position.update(position_params)
+        render json: @position
+      else
+        render json: @position.errors, status: :unprocessable_entity
+      end
+    end
+
+    # DELETE /positions/1
+    def destroy
+      @position.destroy
+      if @position.destroy
+        head :no_content, status: :ok
+      else
+        render json: @position.errors, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    # Use callbacks to share common setup or constraints between actions.
+    def set_position
```

Comment:
Indentation for these methods?

Reply:
don't think they should be indented? rubocop is happy with it